public/locales/en-US/send.ftl

```diff
@@ -73,7 +73,17 @@ fileCount = { $num ->
     [one] 1 file
    *[other] { $num } files
 }
-# size is a localized number followed by a unit of bytes, ex. 2.5GB
+# byte abbreviation
+bytes = B
+# kibibyte abbreviation
+kb = KB
+# mebibyte abbreviation
+mb = MB
+# gibibyte abbreviation
+gb = GB
+# localized number and byte abbreviation. example "2.5MB"
+fileSize = { $num }{ $units }
+# size is a fileSize string
```

Comment:
Size is the size of the file, displayed using the fileSize message as format (e.g. "2.5MB")